tests/scripts/create-dev-cluster.sh

```diff
@@ -4,35 +4,39 @@ DEFAULT_NS="rook-ceph"
 CLUSTER_FILES="common.yaml operator.yaml cluster-test.yaml cluster-on-pvc-minikube.yaml dashboard-external-http.yaml toolbox.yaml"
 MONITORING_FILES="monitoring/prometheus.yaml monitoring/service-monitor.yaml monitoring/exporter-service-monitor.yaml monitoring/prometheus-service.yaml monitoring/rbac.yaml"
 SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
-ROOK_EXAMPLES_DIR="${SCRIPT_ROOT}/../../deploy/examples/"
+
+# Script arguments: new arguments must be added here (following the same format)
+export ROOK_PROFILE_NAME="${ROOK_PROFILE_NAME:=rook}" ## Specify the minikube profile name
+export ROOK_CLUSTER_NS="${ROOK_CLUSTER_NS:=$DEFAULT_NS}" ## CephCluster namespace
+export ROOK_OPERATOR_NS="${ROOK_OPERATOR_NS:=$DEFAULT_NS}" ## Rook operator namespace (if different from CephCluster namespace)
+export ROOK_EXAMPLES_DIR="${EXAMPLES_DIR:=deploy/examples}" ## Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)
```

Comment:
just for consistency..

```suggestion
export ROOK_EXAMPLES_DIR="${ROOK_EXAMPLES_DIR:=deploy/examples}" ## Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)
```

Reply:
fixed